### DISCUSSION TITLE
Next Dev Session (19 February !!!Time Change!!!) ALL ARE WELCOME

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1134] [General]
We would like to change to the below times (30 minutes earlier) Please let us know if this works. Everyone is welcome and please feel free to add agenda items! (@rht and @EwoutH )

When: 19 February 1230 GMT/0730 EST/2030 Singapore

Where: [via Jitsi](https://www.google.com/url?q=https://meet.jit.si/ThinOilsConfuseWearily&sa=D&source=calendar&usd=2&usg=AOvVaw0_kKttuXPY3O2p69SBh72c)

Agenda: 
 * Release
     * Can we create a release page on [GitHub releases](https://github.com/projectmesa/mesa/releases) with the 0.9.0 change log (or linking to it)? This makes the update more visible for users.
     * https://github.com/projectmesa/mesa/issues/1076 should be addressed for users wanting so see actual code changes between their versions and 0.9.0.
 * Google Summer of Code 
 * Numfocus #1096 
 * Tutorial updates
     * #1148 
     * Update tutorial links from google groups (can't find the issue or discussion)  
  * #1155

**Comment by EwoutH** [comment]
Awesome, noted! I would like to put an item on the agenda: [Participation in the Google Summer of Code program](https://github.com/projectmesa/mesa/discussions/1140).

**Reply by tpike3** [reply]
Excellent, thanks for bringing this up it is on the agenda.

**Comment by tpike3** [comment]
@rht @EwoutH @jackiekazil @dmasad and anyone else who is interested, Don't forget Dev meeting tomorrow 19 February 1230 GMT/0730 EST/2030 Singapore
